### PR TITLE
Add install support for other Unixes

### PR DESCRIPTION
**Pull request by J5lx**
This is an extremely trivial change, but I would nonetheless like a quick review of the macOS-specific part because I’m not an Apple user.

### CODE REVIEW

File: app/app.pro [hunk 1]
```diff
@@ -164,7 +164,7 @@ win32 {
     RC_FILE = data/pencil2d.rc
 }
 
-linux {
+unix {
```
Comment (CandyFace) [inline app/app.pro:167]: I'm fairly certain this should be
```suggestion
unix:!macx
```

Because Mac OS uses other conventions which are setup above.

Comment (J5lx) [reply app/app.pro:167]: @scribblemaniac I thought so too, but scribblemaniac suggested doing it this way, see https://github.com/pencil2d/pencil/issues/1127#issuecomment-440585025.

Comment (CandyFace) [reply app/app.pro:167]: Hmm.. ah maybe that's true... alright then, no further comments. Looks fine to me.

**Comment by J5lx** [comment]
I got more reviews than I asked for and no unresolved discussions, so I’m merging this.

**Pull request merged**